pkgs/tools/security/wordlists/dirbuster.nix

```diff
@@ -0,0 +1,29 @@
+{ fetchurl
+, stdenv
+, unzip
+}:
+
+stdenv.mkDerivation  rec {
+  pname = "DirBuster";
+  version = "1.0-RC1";
+
+  src = fetchurl {
+    url = "mirror://sourceforge/dirbuster/DirBuster%20%28jar%20%2B%20lists%29/${version}/${pname}-${version}.zip";
+    sha256 = "sha256-2oDRe9NjvGDT5yFqPEMylhfL1iCsVeVeJ1G9MXfQnqE=";
+  };
+
+  unpackCmd = "${unzip}/bin/unzip $curSrc";
+
+  dontConfigure = true;
+
+  installPhase = ''
+    mkdir -p $out/share/dirbuster
+    install -m 444 directory-list-*.txt $out/share/dirbuster/
+  '';
+
+  meta = with stdenv.lib; {
+    homepage = "https://sourceforge.net/projects/dirbuster/";
+    license = licenses.gpl2;
+    maintainers = with maintainers; [ pamplemousse ];
+  };
+}
```

Comment:
at lot of these could probably converted over to a `fetchzip`, and just use `postFetch` to unpack the correct contents

Reply:
Do you mean `postUnpack`?

I just tried using `postUnpack`, and it seems that `$out` is not yet present...
What about I just replace the `fetchurl` by a `fetchzip`, and get rid of the `unpackCmd`?

Reply:
no, `postFetch`

it's special to fetchurl, which is used by fetchzip and fetchFromGit{Hub,Lab}